plugins/modules/parallels_desktop.py

```diff
@@ -0,0 +1,685 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright Sviatoslav Sydorenko
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""Support for making sure that Parallels Desktop stays dead."""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {  # noqa: WPS407
+    'metadata_version': '1.1',
+    'status': [
+        'preview',
+    ],
+    'supported_by': 'community',
+}
+
+
+DOCUMENTATION = """
+---
+
+module: parallels_desktop
+version_added: 2.3.0
```

Comment:
@samdoran this was written with the assumption that the minor version would be incremented by 1. But since it got incremented by 3, should it be updated here?

Reply:
I'll see about rectifying this.

Reply:
Fixed.